x-pack/plugins/uptime/public/components/common/higher_order/responsive_wrapper.test.tsx

```diff
@@ -20,12 +20,12 @@ describe('ResponsiveWrapper HOC', () => {
   });
 
   it('renders a responsive wrapper', () => {
-    const component = shallowWithIntl(<WrappedByHOC isResponsive={true} />);
-    expect(component).toMatchSnapshot();
+    const { getByTestId } = render(<WrappedByHOC isResponsive={true} />);
```

Comment:
Opted to refactor these tests, however there were more complex tests in `monitor_list.test.jsx` that I opted not to refactor at this time.

Reply:
Thanks for taking the initiative here!